makefiles/cflags.inc.mk

```diff
@@ -29,9 +29,11 @@ endif
 # disabled by the user
 CC_NOCOLOR ?= 0
 ifeq ($(CC_NOCOLOR),0)
+ ifneq ($(TERM),dumb)
```

Comment:
You could set `CC_NOCOLOR` to `1` if a dumb term is detected. You could probably do that at around line 30-31. No need to do another wrap around this block,